tests/console/sshfs.pm

```diff
@@ -1,38 +1,36 @@
 # SUSE's openQA tests
 #
 # Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2016 SUSE LLC
+# Copyright © 2012-2017 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
 # Summary: Mount and access a directory using sshfs
-# Maintainer: Stephan Kulow <coolo@suse.com>
+# Maintainer: Oliver Kurz (okurz@suse.de>
 
 use base "consoletest";
 use strict;
 use testapi;
+use utils 'zypper_call';
 
 sub run() {
     select_console 'root-console';
-
-    assert_script_run("zypper -n in sshfs");
+    zypper_call('in sshfs');
     script_run('cd /var/tmp ; mkdir mnt ; sshfs localhost:/ mnt', 0);
-    assert_screen "accept-ssh-host-key";
+    assert_screen 'accept-ssh-host-key';
     type_string "yes\n";    # trust ssh host key
     assert_screen 'password-prompt';
     type_password;
-    send_key "ret";
+    send_key 'ret';
     assert_screen 'sshfs-accepted';
-    script_run('cd mnt/tmp');
-    assert_script_run("zypper -n in xdelta3");
```

Comment:
oh, not arguing that at all - the test case is absolutely valid and would be a shame to be lost. We should just find a way to make it more obvious as the team grows - so that we don't have to remember the special cases, but can read them up in the description of the test (and maybe even from the name of the test case)